justfile

```diff
@@ -0,0 +1,103 @@
+# List available commands
+_default:
+  just --choose --chooser "fzf +s -x --tac --cycle"
+
+clean:
+    #!/usr/bin/env bash
+    rm -rf ios/build
+    rm -rf android/app/src/main/assets/Database
+    rm -rf ios/NativeSigner/Database
+    cd rust
+    cargo clean
+
+switch_to_android:
+    #!/usr/bin/env bash
+    cp rust/os-specific-lock/android/Cargo.lock rust/
+    cp rust/signer/android-hack/Cargo.toml rust/signer/
+
+switch_to_ios:
+    #!/usr/bin/env bash
+    cp rust/os-specific-lock/ios/Cargo.lock rust/
+    cp rust/signer/ios-hack/Cargo.toml rust/signer/
+
+build_android:
+    #!/usr/bin/env bash
+    cd scripts
+    ./build.sh android
+    cd ..
+
+build_ios:
+    #!/usr/bin/env bash
+    cd scripts
+    ./build.sh ios
+    cd ..
+
+# build all - this is probably impossible
```

Comment:
it's possible https://cheatography.com/linux-china/cheat-sheets/justfile/
![image](https://user-images.githubusercontent.com/17856421/156026001-55120705-42db-44f9-96fd-8748086fb949.png)

Reply:
yes, these commands could be called of course, but we do not have a toolchain combination that will really build both on any platform yet. iOS build requires macos for which we havent's established or tested any android toolchain at the moment. Hopefully this will change.